cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -125,6 +126,7 @@
 
     Set<String> load = new TreeSet<>(String.CASE_INSENSITIVE_ORDER);
     load.add(TIME_PARAM);
+    load.add(START_MS_PARAM);
```

Comment:
I think we should 
1. keep `END_MS_PARAM`, and
2. sanity check mutual exclusion of `end` and `time` parameter.

Sorry if my previous comment was not clear.